src/deriver/tom/long_term.py

```
@@ -0,0 +1,209 @@
+
+MAX_FACT_DISTANCE = 0.85
+
+@ai_track("Tom Inference")
+@observe(as_type="generation")
+async def get_user_representation_long_term(
+    chat_history: str, 
+    session_id: str,
+    embedding_store: CollectionEmbeddingStore,
+    user_representation: str = "None", 
+    tom_inference: str = "None", 
+    facts: Optional[List[str]] = None,
```

Ruff (0.8.2)

Undefined name `Optional`

(F821)